packages/gatsby-source-wordpress/src/gatsby-browser.ts

```diff
@@ -4,8 +4,18 @@ import ReactDOM from "react-dom"
 
 let hydrateRef
 let isFirstHydration = true
+const isSafari = /^((?!chrome|android|crios|fxios).)*safari/i.test(
```

Comment:
I believe `crios` is what is preventing Chrome iOS (Safari) from identifying it as Safari in user agent, will it be able to verify if it's an iPhone from user agent?

https://chromium.googlesource.com/chromium/src/+/HEAD/docs/ios/user_agent.md

Reply:
Oh good call. As far as I'm aware all IOS browsers are safari in disguise, so we should be able to move crios/fxios over next to safari to fix it.